.github/workflows/tests.yaml

```diff
@@ -12,6 +12,11 @@ jobs:
       - name: "💬 Set matrix output"
         id: set-matrix
         run: |
+          echo "${OSTYPE}"
+          echo "${OSTYPE}"
+          echo "${OSTYPE}"
+          echo "${OSTYPE}"
+          echo "${OSTYPE}"
```

Comment:
If this is intended, a comment might be good to explain why the duplication :-)

Reply:
nope, I just rebased my commits, but IIRC the commit message said it was for testing.

Removed.